warehouse/routes.py

```diff
@@ -36,6 +38,11 @@ def includeme(config):
         traverse="/{name}/{version}",
     )
     config.add_route("packaging.file", "/packages/{path:.*}")
+    config.add_route(
+        "packaging.search",
```

Comment:
I'd rather just leave the search bit out for this PR and leave it focused on the index view, the search box can be left non functional for now.